<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>raporlar</class>
 <widget class="QWidget" name="raporlar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 87 22pt &quot;Segoe UI Black&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>SÖZLEŞME SAYISI</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
font: 87 22pt &quot;Segoe UI Black&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>ARAÇ SAYISI</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 87 22pt &quot;Segoe UI Black&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>MÜŞTERİ SAYISI</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 87 22pt &quot;Segoe UI Black&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>CİRO TOPLAM</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLCDNumber" name="lcdNumber_ARACSAYISI"/>
       </item>
       <item row="2" column="0">
        <widget class="QLCDNumber" name="lcdNumber_SOZLESMESAYISI"/>
       </item>
       <item row="1" column="0">
        <widget class="QLCDNumber" name="lcdNumber_MUSTERISAYISI"/>
       </item>
       <item row="3" column="0">
        <widget class="QLCDNumber" name="lcdNumber_CIROTOPLAM"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
